set.mm

```diff
@@ -484868,6 +484888,19 @@ its graph has a given second element (that is, function value).
     cun simp3 syl3anc eqtrid ) CAEZDBEZABFZGHZIZCDRZBJDCRZBJZDUGUHBCDKLUFUCUBBA
     FZGHUIDHUBUCUEMUBUCUENUFUJUDGABOUBUCUESPBADCQTUA $.
 
+  ${
+    rnexd.1 $e |- ( ph -> A e. V ) $.
+    $( The range of a set is a set.  Deduction version of ~ rnexd (Contributed
```

Comment:
```suggestion
    $( The range of a set is a set.  Deduction version of ~ rnex .
       (Contributed
```

Period also missing in: imaexd, ghmquskerlem2

Reply:
Periods added.